### PR TITLE
updating sdk LCD-79

### PR DESCRIPTION
**Pull request by rafaelTelesS**
## What does PR do?

The TagoIO Sdk has been updated to work with the new access rule system for analysis. This system allows you to control who can access your analysis, what actions they can perform, and under what conditions. To use the new system, you need to update your Sdk version and follow the documentation on how to create and manage access rules. 

## Type of alteration

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Documentation update

### CODE REVIEW

**Review by vitorfdl — CHANGES_REQUESTED** [review]
As you are already updating the scripts, also make sure you update:

- Remove context.log or change to console.info, console.debug, etc..

File: README.md [hunk 1]
```diff
@@ -2,14 +2,12 @@
 This analysis retrieves the device list of your account and print to the console.
```
Comment (Freddyminu) [inline README.md]: This analysis retrieves a list of devices from your account and prints it to the console.

File: README.md [hunk 2]
```diff
@@ -2,14 +2,12 @@
...
 
-Steps to generate an account_token:<br>
-1 - Enter the following link: https://admin.tago.io/account/<br>
-2 - Select your Profile.<br>
-3 - Enter Tokens tab.<br>
-4 - Generate a new Token with Expires Never.<br>
-5 - Press the Copy Button and place at the Environment Variables tab of this analysis.<br>
+
+Steps to add a new policy:
+   1 - Click in the button "Add Policy" in this url: https://admin.tago.io/am;
```
Comment (Freddyminu) [inline README.md]: 1 - Click the button "Add Policy", at this url: https://admin.tago.io/am;

File: README.md [hunk 3]
```diff
@@ -2,14 +2,12 @@
...
+   2 - In the Target select Analysis with the field Id and choose your analysis in the list;
```
Comment (Freddyminu) [inline README.md]: 2 - In the Target selector, select the Analysis with the field set as "ID" and choose your Analysis in the list;

File: README.md [hunk 4]
```diff
@@ -2,14 +2,12 @@
...
+   3 - Click in "Click to add a new permission" and select "Device" with rules "Access" with the field "Any";
```
Comment (Freddyminu) [inline README.md]: 3 - Click the  "Click to add a new permission" element and select "Device" with the rule "Access" and with the field as "Any";

File: README.md [hunk 5]
```diff
@@ -2,14 +2,12 @@
...
+   4 - To finish click on save;<br>
```
Comment (Freddyminu) [inline README.md]: 4 - To save your new Policy, click the save button in the bottom right corner;<br>

File: analysis.js [hunk 6]
```diff
@@ -3,35 +3,21 @@
  ** Get Device List
  **
  ** This analysis retrieves the device list of your account and print to the console.
```
Comment (Freddyminu) [inline analysis.js]: ** This analysis retrieves a list of devices from your account and prints it to the console.

File: analysis.js [hunk 7]
```diff
@@ -3,35 +3,21 @@
...
- ** 2 - Select your Profile.
- ** 3 - Enter Tokens tab.
- ** 4 - Generate a new Token with Expires Never.
- ** 5 - Press the Copy Button and place at the Environment Variables tab of this analysis.
+ ** How to use:
+ ** To analysis works, you need to add a new policy in your account. Steps to add a new policy:
+ **  1 - Click in the button "Add Policy" in this url: https://admin.tago.io/am;
+ **  2 - In the Target select Analysis with the field Id and choose your analysis in the list;
+ **  3 - Click in "Click to add a new permission" and select "Device" with rules "Access" with the field "Any";
+ **  4 - To finish click on save;
```
Comment (Freddyminu) [inline analysis.js]: so colar as correções do readme para aqui ja que so ta repetindo de la

**Comment by Freddyminu** [comment]
Olhei o read me e ta meio estranho, acho que faltou quebra de linha: 

image

**Review by Freddyminu — APPROVED** [review]

**Review by vitorfdl — APPROVED** [review]

**Pull request merged**